### PR TITLE
Feat/add-generic-head-component

### PR DESCRIPTION
**Pull request by spences10**
Added a generic head component to help for SEO and reduce repeated information throughout the project

### CODE REVIEW

File: src/routes/projects/[slug].svelte [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <script context="module">
+  import { page } from '$app/stores'
+  import Head from '$lib/components/head.svelte'
   import { client } from '$lib/graphql-client'
   import { projectQuery } from '$lib/graphql-queries'
   import { marked } from 'marked'
+  import { siteMetadataStore } from '../../stores/site-metadata'
```
Comment (notrab) [inline src/routes/projects/[slug].svelte]: Can this not be `$stores/site-metadata`?

File: src/routes/posts/index.svelte [hunk 2]
```diff
@@ -1,7 +1,10 @@
 <script context="module">
+  import { page } from '$app/stores'
+  import Head from '$lib/components/head.svelte'
   import { client } from '$lib/graphql-client'
   import { postsQuery } from '$lib/graphql-queries'
   import { marked } from 'marked'
+  import { siteMetadataStore } from '../../stores/site-metadata'
```
Comment (notrab) [inline src/routes/posts/index.svelte]: Can this not be `$stores/site-metadata`?

File: src/routes/posts/[slug].svelte [hunk 3]
```diff
@@ -1,7 +1,10 @@
 <script context="module">
+  import { page } from '$app/stores'
+  import Head from '$lib/components/head.svelte'
   import { client } from '$lib/graphql-client'
   import { postQuery } from '$lib/graphql-queries'
   import { marked } from 'marked'
+  import { siteMetadataStore } from '../../stores/site-metadata'
```
Comment (notrab) [inline src/routes/posts/[slug].svelte]: Can this not be `$stores/site-metadata`?

File: src/routes/index.svelte [hunk 4]
```diff
@@ -1,7 +1,12 @@
 <script context="module">
+  import Head from '$lib/components/head.svelte'
   import ProjectCard from '$lib/components/project-card.svelte'
   import { client } from '$lib/graphql-client'
   import { authorsQuery, projectsQuery } from '$lib/graphql-queries'
+  import {
+    authorsStore,
+    siteMetadataStore,
+  } from '../stores/site-metadata'
```
Comment (notrab) [inline src/routes/index.svelte]: Can this not be `$stores/site-metadata`?

File: src/routes/about.svelte [hunk 5]
```diff
@@ -1,7 +1,10 @@
 <script context="module">
+  import { page } from '$app/stores'
+  import Head from '$lib/components/head.svelte'
   import { client } from '$lib/graphql-client'
   import { authorsQuery } from '$lib/graphql-queries'
   import { marked } from 'marked'
+  import { siteMetadataStore } from '../stores/site-metadata'
```
Comment (notrab) [inline src/routes/about.svelte]: Can this not be `$stores/site-metadata`?

File: src/lib/components/head.svelte [hunk 6]
```diff
@@ -0,0 +1,53 @@
+<script>
+  import {
+    authorsStore,
+    siteMetadataStore,
+  } from '../../stores/site-metadata'
```
Comment (notrab) [inline src/lib/components/head.svelte:5]: Can this not be `$stores/site-metadata`?

**Review by notrab — CHANGES_REQUESTED** [review]
Left some comments on importing. Following what is done [here](https://github.com/GraphCMS/graphcms-sveltekit-portfolio-and-blog-starter/blob/main/jsconfig.json#L6), these could follow a similar format?

File: src/routes/projects/[slug].svelte (hunk 1 above)
Comment (spences10) [reply src/routes/projects/[slug].svelte]: Jep! I'll get that done, thanks

**Pull request merged**